rotkehlchen/chain/evm/tokens.py

```diff
@@ -155,6 +155,41 @@ def _get_token_balances(
             balances[token] += normalized_balance
         return balances
 
+    def get_staking_contract_balances(
```

Comment:
Questions.

1. Will it also be used for convex, or only for curve?
2. If only for curve, then maybe move contents of this function to the curve-specific place?
3. If for both, then maybe move the function to `ProtocolWithBalance` class or somewhere like that? `evm/tokens.py` sounds too generic to me to put staking-specific functionality here. I might be wrong though.

Reply:
Is used both by convex and curve. I've moved it to a private method